apps/web/src/modules/dao/components/Explore/ExploreMyDaos.tsx

```diff
@@ -39,8 +35,11 @@ export const ExploreMyDaos = () => {
             const bid = dao.highestBid?.amount ?? undefined
             const bidInEth = bid ? ethers.utils.formatEther(bid) : undefined
 
+            if (!dao.chainId) return null
```

Comment:
Would recommend returning some sort of error in this case, but I'm wondering what the likelihood of this happening is.

Reply:
I had chainId as optional at first to keep the type compatible with explore but I think the better solution is to add chainId to the explore query and make it non optional. Just made this change